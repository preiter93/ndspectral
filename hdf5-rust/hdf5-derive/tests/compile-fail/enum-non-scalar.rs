extern crate hdf5_derive;
use hdf5_derive::H5Type;

#[derive(H5Type)]
//~^ ERROR proc-macro derive
//~^^ HELP H5Type can only be derived for enums with scalar discriminants
enum Foo {
    Bar,
}

fn main() {}
