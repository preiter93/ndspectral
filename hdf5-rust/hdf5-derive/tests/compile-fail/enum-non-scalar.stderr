error: proc-macro derive panicked
 --> $DIR/enum-non-scalar.rs:4:10
  |
4 | #[derive(H5Type)]
  |          ^^^^^^
  |
  = help: message: H5Type can only be derived for enums with scalar discriminants
